tests/ignore_comments/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+// Superb function
+fn main() {
+    // Superb variable
+    let a = 10;
+}
+
+// Superb struct
+struct A {
+    // Superb field
+    a: i32,
+}
+
+// Superb enum
+enum B {
+    // Superb variant
+    A,
+}
+
+// Superb trait
+trait C {
+    // Superb method
+    fn a(&self);
+}
+
+// Superb impl
+impl C for A {
+    // Superb method
+    fn a(&self) {
+        // Superb expression
+        let a = 10;
```

Comment:
I think this test case is inadvertantly exposing a sever limitation of the current representer. This function and its local variable have the same name and are assigned the same placeholder. That should not be the case however, as they refer to different items.

For running the representer in production, this means that two identical solutions may not receive the same representation, simply because two two different items shared or didn't share the same name.

Reply:
(I extracted this into a separate issue https://github.com/exercism/rust-representer/issues/43)